src/app/components/core/notifications/notification-service-modal/notification-service-modal.component.html

```diff
@@ -0,0 +1,79 @@
+<div id="notification-service-modal" class="modal">
+    <div class="modal-background"></div>
+    <div class="modal-card">
+        <header class="modal-card-head">
+            <p *ngIf="isNotificationServiceAvailable" class="modal-card-title is-size-5">{{name}} Notification Service</p>
+            <p *ngIf="!isNotificationServiceAvailable" class="modal-card-title is-size-5">Notification Service</p>
+            <button class="delete" aria-label="close" (click)="toggleModal(false)"></button>
+        </header>
+        <section class="modal-card-body">
+            <form name="notification-form" id="notificationForm" #fg="ngForm"
+                (ngSubmit)="fg.form.valid && saveChanges()" novalidate>
+                <div *ngIf="!isNotificationServiceAvailable" class="columns is-mobile">
+                    <div class="column">
+                        <div class="field is-horizontal">
+                            <div class="field-label has-text-left">
+                                <label class="label">Name</label>
+                            </div>
+                            <div class="field-body">
+                                <div class="field">
+                                    <input *ngIf="!isNotificationServiceAvailable" type="text" class="input is-small"
+                                        name="name" [(ngModel)]="name" placeholder="FogLAMP Notifications" required
+                                        trim="blur" />
+                                    <small *ngIf="fg?.controls['name']?.invalid"
```

Comment:
`<small *ngIf="fg?.controls['name']?.dirty && fg?.controls['name']?.invalid "
                                        class="help is-danger level-left">*required</small>` to hide required field check on page load.

Reply:
It doesn't work, if form is submitted with blank input value of name (without touching name field)

Reply:
`if (!this.form.valid) {
      this.form.controls['name'].markAsTouched();
      return;
    }` 
use this check inside proxy() method to fix (without touching the field issue)